todo/todo_test.go

```diff
@@ -2,48 +2,88 @@ package todo
 
 import (
 	"fmt"
+	"os"
+	"practice/go-todo-list/config"
+	"practice/go-todo-list/db"
 	"testing"
 
 	"github.com/stretchr/testify/require"
 )
 
 const (
 	task         = "some random testing task"
-	taskID       = 48
+	taskID       = 1
 	updateTask   = "updated testing task"
-	updateTaskID = 50
-	deleteTaskID = 51
+	updateTaskID = 1
+	deleteTaskID = 1
 )
 
 func TestCreate(t *testing.T) {
-	taskID, err := Create(task)
-	fmt.Println("b ", taskID)
-	require.NotNil(t, taskID)
+	os.Setenv("ENVIRONMENT", "test")
+	config.Load()
+	db := db.InitDB()
+	task_id, err := Create(task)
 	require.NoError(t, err)
+
+	fmt.Println(task_id)
+
+	var task_check string
+	s, err := db.Prepare("SELECT task AS task_check FROM todo_table WHERE task_id = $1")
+	rows := s.QueryRow(task_id)
+	rows.Scan(&task_check)
+
+	require.Equal(t, task, task_check)
```

Comment:
We are asserting this. It should be `assert`

Reply:
Done.